README.md

```diff
@@ -52,6 +52,7 @@ npm install --save-dev @reportportal/agent-js-playwright
 | skippedIssue          | *Default: true.* ReportPortal provides feature to mark skipped tests as not 'To Investigate' items on WS side.<br> Parameter could be equal boolean values:<br> *TRUE* - skipped tests considered as issues and will be marked as 'To Investigate' on Report Portal.<br> *FALSE* - skipped tests will not be marked as 'To Investigate' on application. |
 | debug                 | This flag allows seeing the logs of the client-javascript. Useful for debugging.                                                                                                                                                                                                                                                                        |
 | includeTestSteps      | *Default: false.* Allows you to see the test steps at the log level.                                                                                                                                                                                                                                                                                    |
+| isCodeRefEnabled      | *Default: false.* Allows you to use Playwrite project name during code ref calculation.                                                                                                                                                                                                                                                                                    |
```

Comment:
The code ref calculation is enabled by default. This feature adds an ability to include playwright project name to it.
So please rename for example to meaningful `includePlaywrightProjectNameToCodeReference`.
Also update the description:
>
Includes Playwright project name to code reference. See [`testCaseId` and `codeRef` calculation](#setTestCaseId). It may be useful when you want to see the different history for the same test cases within different projects.